### DISCUSSION TITLE
RSS Cadence and excludeRecentSearch

### DISCUSSION THREAD

**Discussion by carried-away** [cross-seed/cross-seed#539] [Q&A] [answered]
I understand that excludeRecentSearch is specifically for search cadence, but does it effect RSS Cadence as well?

For example, if I have excludeRecentSearch set to 1 year. I download a TV show that just aired and got posted in tracker 1, it completes and cross-seed searches for that torrent in tracker 2, however the torrent doesn't get uploaded to tracker 2 for another 5 minutes. Will the torrent from tracker 1 no longer be searched in RSS Cadence for another year?

**Comment by zakkarry** [comment, marked as answer]
No it will match on rssCadence regardless of excludeRecentSearch

**Reply by carried-away** [reply]
Thank you
